src/Compilers/CSharp/Test/Semantic/Semantics/SemanticErrorTests.cs

```diff
@@ -19951,11 +19951,6 @@ public void Func(object o, string p)
         table[p] = o;
     }
 }
-
-abstract class Foo
-{
-    public int x; // no diagnostic when in abstract class (for no apparent reason except that's the way Dev10 does it)
```

Comment:
So Dev11 broke compat with Dev10, where it was restored in Dev11, broken again in Dev14, and will be reverted in Dev15?